### PR TITLE
Grafana setup

### PR DESCRIPTION
**Pull request by GitBluub**

### CODE REVIEW

File: back/src/main.ts [hunk 1]
```diff
@@ -2,10 +2,43 @@ import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
 import { PrismaService } from './prisma/prisma.service';
 import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
-import { ValidationPipe } from '@nestjs/common';
+import { CallHandler, ExecutionContext, Injectable, NestInterceptor, ValidationPipe } from '@nestjs/common';
+import {RequestLogger, RequestLoggerOptions} from 'json-logger-service';
+import { tap } from 'rxjs';
+
+@Injectable()
```
Comment (zoriya) [inline back/src/main.ts:9]: There are a mix of spaces and tabs for indentation. run prettier or something

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -54,3 +64,190 @@ services:
...
+  gateway:
+    image: nginx:1.25.1
+    depends_on:
+      - read
+      - write
+    entrypoint:
+      - sh
+      - -euc
+      - |
+        cat <<EOF > /etc/nginx/nginx.conf
```
Comment (zoriya) [inline docker-compose.dev.yml:195]: why is this inlined when you already created the same file?

**Review by zoriya — COMMENTED** [review]
Is graphana lightweight? I'm not sure I want to run graphana with the dev dockerfile by default.

File: back/src/main.ts (hunk 1 above)
Comment (GitBluub) [reply back/src/main.ts:15]: ran prettier

**Pull request merged**